sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,104 @@
+| Indonesian | Aris Cahyadi Risdianto ([@ariscahyadi](https://github.com/ariscahyadi)) | https://kubernetes.io/id/ | [#kubernetes-docs-id](https://kubernetes.slack.com/archives/CJ1LUCUHM) |
+| Italian | Fabrizio Pandini ([@fabriziopandini](https://github.com/fabriziopandini)) | https://kubernetes.io/it/ | [#kubernetes-docs-it](https://kubernetes.slack.com/archives/CGB1MCK7X) | 
+| Japanese | Masahiro Kitamura ([@nasa9084](https://github.com/nasa9084)), Kohei Ota ([@inductor](https://github.com/inductor)) | https://kubernetes.io/ja/ | [#kubernetes-docs-ja](https://kubernetes.slack.com/archives/CAG2M83S8) | 
+| Korean | Seokho Son ([@seokho-son](https://github.com/seokho-son)) | https://kubernetes.io/ko/ | [#kubernetes-docs-ko](https://kubernetes.slack.com/archives/CA1MMR86S) | 
+| Portuguese | Ricardo Katz ([rikatz](https://github.com/rikatz)), Mauren Berti ([@stormqueen1990](https://github.com/stormqueen1990)), Edson C ([@edsoncelio](https://github.com/edsoncelio)) | https://kubernetes.io/pt-br/ | [#kubernetes-docs-pt](https://kubernetes.slack.com/archives/CJ21AS0NA) | 
+| Russian | Dmitry Shurupov ([@shurup](https://github.com/shurup)) | https://kubernetes.io/ru/ | [#kubernetes-docs-ru](https://kubernetes.slack.com/archives/CPZ9KD9TN) |
+| Spanish | Rael Garcia ([@raelga](https://github.com/raelga)), Victor Morales ([@electrocucaracha](https://github.com/electrocucaracha/)) | https://kubernetes.io/es/ | [#kubernetes-docs-es](https://kubernetes.slack.com/archives/CH7GB2E3B) | 
+| Ukrainian | Maksym Vlasov ([@MaxymVlasov](https://github.com/MaxymVlasov)) | https://kubernetes.io/uk/ | [#kubernetes-docs-uk](https://kubernetes.slack.com/archives/CSKCYN138) | 
+| Vietnamese | | https://kubernetes.io/vi/ | [#kubernetes-docs-vi](https://kubernetes.slack.com/archives/CPHAWNF1Q) |  
+
+"In progress" teams have started localization work, but are not yet live on the Kubernetes site.
+
+## Localization team structure
+
+Anyone is open to start a new localization effort for any language or join an ongoing localization team. If you are interesting in learning more about specific localization requirements or processes see the [Localization Contributing Guide](https://kubernetes.io/docs/contribute/localization/).  
+
+Localization team are self sufficient communities within SIG Docs. Localization teams are responsible for
+
+- Localizing content on kuberntes.io into their chosen language.  
+- Timely reviews and merging PRs for their languages according to the [review and merge prcesses](https://kubernetes.io/docs/contribute/review/) of the kubernetes/website repo.
+- Building a community that can support their localization efforts. Note you need at least two contributors to begin a localization team because contributors can't approve their own pull requests. You will likely need more contributors, reviewers, and approvers to create a sustainable localization effort.
```

Comment:
```suggestion
- Building a community that can support their localization efforts. Note that you need at least two contributors to begin a localization team because contributors can't approve their pull requests. You will likely need more contributors, reviewers, and approvers to create a sustainable localization effort.
```

Reply:
updated